bedrock/firefox/templates/firefox/fights-for-you.de.html

```diff
@@ -36,7 +36,9 @@
     title='Firefox Fights For You',
     desc='Privatsphäre sollte einfach sein. Daten sicher. Einstellungen eindeutig. Wir kämpfen für ein besseres Internet – mit Produkten, die Dir auch online die Kontrolle über Dein Leben geben.',
     class='mzp-has-image mzp-t-firefox mzp-t-product-firefox mzp-t-dark',
+    include_cta=True
   ) %}
+  <a id="fffy-video-button" href="https://www.youtube.com/watch?v=3NG6dZl9kpA" data-id="3NG6dZl9kpA" class="mzp-c-button mzp-t-secondary mzp-t-dark mzp-t-small" data-link-type="button" data-link-name="Watch video">Video anschauen</a>
```

Comment:
I'm not a designer, but fwiw at first I missed the "watch video" button altogether. It could perhaps do with being a bit larger, as opposed to using `mzp-t-small`?

Reply:
This is the component the designer used.